governance-aepp/src/theme/ae-button.scss

```diff
@@ -0,0 +1,81 @@
+.ae-button.round[data-v-d0360416], .ae-button.flat[data-v-d0360416] {
```

Comment:
don't use component if you need to completely restyle

Reply:
it's partially restyled, the main style is a lot bigger and too specific